src/main/webapp/filterScript.js

```diff
@@ -0,0 +1,43 @@
+const UNCLICKED = 'unclicked-filter';
```

Comment:
nit: no need to include `Script` in the file name since the `.js` extension already says that it's a script.